### ISSUE TITLE
Backspace alpha-numeric boundary fails on unicode

### ISSUE THREAD

**Issue by bumbread** [4coder-editor/4coder#304]
Version: 4.1.7

**Repro Steps (Including a snippet of any potentially related customization code)**

1. In any file type the following code and place the cursor where the `|` symbol is located (without typing the pipe symbol)
```
function something {
  абвгд|
```
2. Type ctrl-backspace

**Description of the Result**

Expectation is that the following code will remain, with cursor at the place of the `|` symbol:
```
function something {
  |
```

However the actual result is the following:
```
function |
```

In other words, for some reason it erases too much

**Additional notes**

Similar bug happens when typing `ctrl-delete`.

**System**
OS: *Windows 10*

**Comment by 4coder-editor** [comment]
I needed to switch the boundary function to using the `_unicode` version of the character set. Also applied this fix to the "delete boundary" command. Fix in 4.1.8
